_pages/how-we-work/software.md

```diff
@@ -22,21 +22,21 @@ If the name pops up, it means GSA has already purchased it. If your software is
 
 _Note: If you're looking for MS Office, see [this page]({{site.baseurl}}/office/)._
 
-## Purchase new software
+# Purchase new software
 
-If the software is not in the TTS inventory or GSA IT Catalog (or if there are no licenses available), the TTS Tech Portfolio requests that you use the [TTS Tech Portfolio General Request Form](https://forms.gle/wPrCXJqDgaksXRwV8) for a _New Software (consult)_.
+Here at TTS, we want to stay current and take advantage of as many emergign technologies as possible. We believe that getting the software you need to do your job should be difficult, time-consuming or stressful. If the software is not in the TTS inventory or GSA IT Catalog (or if there are no licenses available), the TTS Tech Portfolio will work with you to meet your needs as best as possible
 
-### Step 1: find or get an ATO
+### Step 1: If you haven't already used the [TTS Tech Portfolio Software Request Form](https://forms.gle/w3MqKNMmLj1vbpWv8) to get a _New Software (consult)_ please do so.
```

Comment:
This section is going to conflict with the PR i made to remove mentions of the General Request form https://github.com/18F/handbook/pull/2093